src/common/BaseAnnotator.ts

```diff
@@ -97,8 +101,16 @@ export default class BaseAnnotator extends EventEmitter {
             this.containerEl.classList.remove(CSS_CONTAINER_CLASS);
         }
 
-        if (this.annotatedEl) {
-            this.annotatedEl.classList.remove(CSS_LOADED_CLASS);
+        const annotatedElement = this.annotatedEl;
+
+        if (annotatedElement) {
```

Comment:
Still need to perform an existence check (can't just do `if (!this.annotatedEl) { return; }` because we still need the rest of the code below the execute).